pom.xml

```diff
@@ -129,6 +129,11 @@
                 <artifactId>commons-net</artifactId>
                 <version>3.6</version>
             </dependency>
+            <dependency>
+                <groupId>xml-apis</groupId>
+                <artifactId>xml-apis</artifactId>
+                <version>1.4.01</version>
```

Comment:
Needed to resolve build error:

Require upper bound dependencies error
+-org.jenkins-ci.main:jenkins-test-harness:2.52
+-org.jenkins-ci.main:jenkins-test-harness-htmlunit:2.31-2
+-xml-apis:xml-apis:1.4.01